Makefile

```diff
@@ -274,6 +274,15 @@ build/$(TEST_JAR): $(TEST_SOURCES) build/$(CONVERTER_JAR)
 	$(JAVAC) -source $(JAVA_TARGET) -target $(JAVA_TARGET) -Xlint:-options -cp "build/jar/*:build/converter/*" -d build/test $(TEST_SOURCES)
 	$(JAR) cf $@ -C build/test .
 
+.ONESHELL:
+cpp-lint:
+	@LINT_SOURCES=$$(ls -1 src/*.cpp src/*/*.cpp | grep -v rustDemangle.cpp)
```

Comment:
Can we use "``" so that we don't need to escape `$`?

Reply:
Sure: 6bcd07823c128eb3d91c2d53b1cb31aee924c729